### PR TITLE
US35630 - Roll Consul and Vault changes forward into Swagger and RBAC Python examples.

### PR DESCRIPTION
**Pull request by hagraham**

### CODE REVIEW

**Review by mtsizis — APPROVED** [review]

File: python-hello-world-service-6/helloworld.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  port: "8200"                           # Bound to env var SPRING_CLOUD_VAULT_PORT at runtime.
+  token: "s.l06JWgzNUHCwZCXM124iyIXr"    # Bound to env var SPRING_CLOUD_VAULT_TOKEN at runtime.
+  cacert: "/etc/ssl/certs/ca-bundle.crt" # Required by MSX.
+
+swagger:
+  rootpath: "/helloworld"             # Required by MSX.
+  secure: true                        # Required by MSX.
+  ssourl: "http://localhost:9515/idm" # CONSUL thirdpartyservices/defaultapplication/swagger.security.sso.baseUrl
+  clientid: "local-public-client"     # CONSUL thirdpartyservices/helloworldservice/public.security.clientId
+  swaggerjsonpath: "swagger.json"     # Required by MSX.
```
Comment (vclkan) [inline python-hello-world-service-6/helloworld.yml:22]: Github says EOL is missing here.

File: python-hello-world-service-6/helloworld.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+#
+# Copyright (c) 2021 Cisco Systems, Inc and its affiliates
+# All Rights reserved
+#
+consul:
```
Comment (vclkan) [inline python-hello-world-service-6/helloworld.yml:5]: nitpick: can we put a space after the header?

File: python-hello-world-service-6/manifest.yml [hunk 3]
```diff
@@ -38,3 +38,23 @@ Containers:
...
+  - Name: "favourite.food"
+    Value: "Pizza"
+  - Name: "favourite.dinosaur"
+    Value: "Moros Intrepidus"
+  - Name: "public.security.clientId"
+    Value: "hello-world-service-public-client"
+
+Secrets:
+  - Name: "secret.squirrel.location"
+    Value: "The acorns are buried under the big oak tree!"
```
Comment (vclkan) [inline python-hello-world-service-6/manifest.yml:60]: same: EOL missing.

**Review by vclkan — APPROVED** [review]
Looking good.
I had a few nitpicks, but nothing preventing it from approval.
We can address them in a follow-up commit too.

**Pull request merged**